frontend/model/contracts/group.js

```diff
@@ -702,6 +702,7 @@ sbp('chelonia/defineContract', {
             createdDate: meta.createdDate,
             groupID: contractID,
             creatorID: innerSigningContractID,
+            isCreator: innerSigningContractID === loggedIn.identityContractID,
```

Comment:
Instead of `isCreator`, I think it'd be better to leave this as is, and add the `creatorID` to the `PROPOSAL_CLOSED` notification. Then, we can check for creator when the notification text is created. (The reason for doing it this way is to keep the contract code as less reliant on external state as possible).